mrmustard/utils/wigner.py

```diff
@@ -30,13 +29,18 @@ def wigner_discretized(rho, qvec, pvec, hbar=settings.HBAR):
         rho (complex array): the density matrix of the state in Fock representation
         qvec (array): array of discretized :math:`q` quadrature values
         pvec (array): array of discretized :math:`p` quadrature values
-        hbar (optional float): the value of `\hbar`, defaults to ``settings.HBAR``.
 
     Retunrs:
         tuple(array, array, array): array containing the discretized Wigner function, and the Q and
             P coordinates (in meshgrid form) in which the function is calculated
     """
+    hbar = settings.HBAR
+    return _wigner_discretized(rho, qvec, pvec, hbar)
+
 
+@njit
+def _wigner_discretized(rho, qvec, pvec, hbar):
```

Comment:
@sylviemonet just a comment about this:

The best thing would be to have 
`@njit`
`def wigner_discretized(rho, qvec, pvec)`:
`    hbar = settings.HBAR`
`    ...`
But unfortunately, `@njit` dislikes `settings.HBAR`. 

This is why I decided to go through the `_wigner_discretized` function, which does take `hbar` as input, but is hidden and won't be used anywhere else. If you have better ideas, ket me know :)